makefiles/tools/programmer.inc.mk

```diff
@@ -8,7 +8,11 @@ endif
 # Don't use the programmer wrapper for the CI (where speed and verbose output
 # are important)
 ifneq (1,$(RIOT_CI_BUILD))
-  USE_PROGRAMMER_WRAPPER_SCRIPT ?= 1
+  ifneq (,$(filter pyocd,$(PROGRAMMER)))
```

Comment:
Maybe add a comment that explains why it's disabled by default with pyocd.

Reply:
The commit message says so but ok.

Reply:
Let me know if the comment is OK